### PR TITLE
Add jmh microbenchmarks to eclipse IDE

### PR DESCRIPTION
**Pull request by bereng**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -1976,6 +1976,7 @@
   <classpathentry kind="src" path="conf" including="hotspot_compiler"/>
   <classpathentry kind="src" output="${test.classes}" path="test/unit"/>
   <classpathentry kind="src" output="${test.classes}" path="test/long"/>
+  <classpathentry kind="src" output="${test.classes}" path="test/microbench"/>
```
Comment (Mmuzaf) [inline build.xml:1979]: `memory` and `burn` paths are missed as well, right? :-)

Comment (bereng) [reply build.xml:1979]: Added along anttasks

Comment (Mmuzaf) [reply build.xml:1979]: Good catch, it seems that `anttask` is missed for the IDEA configuration as well. 
Should we fix it here?
https://github.com/apache/cassandra/blob/trunk/ide/idea-iml-file.xml

Comment (bereng) [reply build.xml:1979]: Apologies for the late reply I have been OOO. I don't have IDEA so I can't help much here... I have added the commit though in hope you or Benjamin can test it.

Comment (Mmuzaf) [reply build.xml:1979]: I've tested your patch - run `ant generate-idea-files` and imported the project from scratch.

Screenshot 2023-08-16 at 14 00 08

**Review by Mmuzaf — APPROVED** [review]

**Review by blerer — APPROVED** [review]

**Pull request closed without merging**